File: frontend/src/app/shared/responses/post.response.ts

export class PostResponse {
  id: number;
  name: string;
  youtubeLink: string;
  date: string;
  content: string;
  active: boolean;
  imagesIds: number[];

  constructor(id: number, name: string, youtubeLink: string, date: string, content: string, active: boolean, imagesIds: number[]) {
    this.id = id;
    this.name = name;
    this.youtubeLink = youtubeLink;
    this.date = date;
    this.content = content;
    this.active = active;
    this.imagesIds = imagesIds;
  }
}

File: src/main/java/com/sgorokh/TimeSaver/controllers/responses/PostResponse.java
package com.sgorokh.TimeSaver.controllers.responses;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;
import java.util.List;

@Builder
@Data
@AllArgsConstructor
public class PostResponse {

    private Long id;
    private String name;
    private String youtubeLink;
    private String date;
    private String content;
    private Boolean active;
    private List<Long> imagesIds;

}
